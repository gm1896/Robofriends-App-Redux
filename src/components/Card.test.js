import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

it('expects to render Card Component', () => {
	expect(shallow(<Card />).length).toEqual(1)
})

it('expects to checkout Snapshot Testing', () => {
	expect(shallow(<Card />)).toMatchSnapshot();
})