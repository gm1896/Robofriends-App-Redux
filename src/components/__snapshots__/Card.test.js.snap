// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to checkout Snapshot Testing 1`] = `ShallowWrapper {}`;
